client/cloudSubnet_service.go

```diff
@@ -31,9 +31,28 @@ func (sm *ServiceManager) CreateCloudSubnet(ip string, cloud_cidr_pool_dn string
 			return nil, err
 		}
 
-		jsonPayload.Array(cloudSubnet.ClassName, "children")
 		jsonPayload.ArrayAppend(zoneCon.Data(), cloudSubnet.ClassName, "children")
 	}
+
+	if cloudRsSubnetToCtx != "" {
+		cloudRsSubnetToCtxJSON := []byte(fmt.Sprintf(`
+		{
+			"cloudRsSubnetToCtx": {
+				"attributes": {
+					"annotation":"orchestrator:terraform",	
```

Comment:
don't hardcode it. Do you have access to the parent orchestrator value?

Reply:
We don't have a definition of the orchestrator in the go client flies. 
"orchestrator:terraform" is used as the annotation in the definition of all the relationship attributes payload.

Reply:
Opened https://github.com/ciscoecosystem/aci-go-client/issues/266 to track this in the future